.github/workflows/tests.yml

```diff
@@ -27,11 +27,11 @@ jobs:
     - name: Fmt
       run: cargo fmt --all -- --check
     - name: Check docs
-      run: cargo doc --no-deps --document-private-items
+      run: cargo doc --all-features --no-deps --document-private-items
       env:
         RUSTDOCFLAGS: "-D warnings"
     - name: Run tests
-      run: cargo test --verbose
+      run: cargo test --all-features --verbose
```

Comment:
I suppose we need to modify `bench` to run storage benchmarks with `iouring`? 🤔

Reply:
added

Reply:
I think we don't want all features (as cryptography will slow down)

Reply:
I think we should explicitly enable the ones we want for benchmarking. So yeah, I think just `--features iouring`.